src/dataloaderinterface/forms.py

```diff
@@ -8,7 +10,14 @@
 from django.db.models.query_utils import Q
 from django.forms.formsets import formset_factory
 
-from dataloaderinterface.models import ODM2User
+from dataloaderinterface.models import ODM2User, SiteAlert
+
+
+class SiteTypeSelect(forms.Select):
+    def create_option(self, name, value, label, selected, index, subindex=None, attrs=None):
+        option = super(SiteTypeSelect, self).create_option(name, value, label, selected, index, subindex, attrs)
+        option['attrs']['title'] = self.choices.queryset.get(name=value).definition if value else ''
```

Comment:
Maybe a try/catch here, depending on the chance we get bad data. Not an issue if option['attrs']['title'] can receive an invalid or null value.

Reply:
I have the `... if value else ''` that puts an empty string in the title if there's no value. It won't crash, but I can do the try/catch to be 120% sure.